rasa/nlu/test.py

```diff
@@ -506,6 +490,40 @@ def evaluate_intents(
     }
 
 
+def try_plot_confusion_matrix(
```

Comment:
why `try_`? How about just `_plot_confusion_matrix` (also making it private)

Reply:
yes, totally forgot the naming policies :(

Reply:
haha 😀If you prefer the `try_` part then I'm also happy to discuss pros and cons!

Reply:
nono, you're totally right 🙂